subsys/net/ip/ipv6.c

```diff
@@ -383,6 +383,11 @@ enum net_verdict net_ipv6_input(struct net_pkt *pkt, bool is_loopback)
 		log_strdup(net_sprint_ipv6_addr(&hdr->src)),
 		log_strdup(net_sprint_ipv6_addr(&hdr->dst)));
 
+	if (net_ipv6_is_addr_unspecified(&hdr->src)) {
+		NET_DBG("DROP: src addr is %s", "unspecified");
```

Comment:
why the format string in this case?

Reply:
> why the format string in this case?

We can share the same string with IPv4 checks.

Reply:
ok, could be done for other drop message on src then

Reply:
> ok, could be done for other drop message on src then

I considered it but they look a bit different in IPv6 side so did not do it in this PR. We can do it separately if needed.

Reply:
sure